### PR TITLE
Dev cli sessions

### PR DESCRIPTION
**Pull request by i-davydenko**
https://github.com/sonic-net/SONiC/pull/1367

Why I did it
Give ability to:
1. configure limit for active login sessions.
2. configure ssh-server / serial console autologout timeout
3. configure sysrq-capabilities (enable / disable)

Work item tracking
Microsoft ADO (number only):

How I did it
Add new service that responsible for serial configuration;
Update existing flows for extended ssh-server configurations in hostcfgd;
Add YANG model to support new configuration.

How to verify it
Which release branch to backport (provide reason below if selected)

Tested branch (Please provide the tested image version)

Description for the changelog
Link to config_db schema for YANG module changes
[ssh_server](https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/doc/Configuration.md#ssh_server)
[serial_console](https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/doc/Configuration.md#serial_console)

A picture of a cute animal (not mandatory but encouraged)
kitty - Cube
image

### CODE REVIEW

**Comment by i-davydenko** [comment]
Hi @lguohan , Hi @qiluo-msft 
Could you please review the changes here and in other related submodules :)

File: src/sonic-yang-models/yang-models/sonic-ssh-server.yang [hunk 1]
```diff
@@ -40,6 +45,20 @@ module sonic-ssh-server {
                         }
                     }
                 }
+                leaf inactivity_timeout {
+                    description "inactivity timeout (in minutes), 0 means no timeout";
+                    default 15;
```
Comment (qiluo-msft) [inline src/sonic-yang-models/yang-models/sonic-ssh-server.yang:50]: >15 [](http://example.com/codeflow?start=28&length=2)

How do you pick the default value? Is it the linux default behavior in old image without this feature?

File: src/sonic-yang-models/yang-models/sonic-serial-console.yang [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    container sonic-serial-console {
+        container SERIAL_CONSOLE {
+            description "SERIAL_CONSOLE part of config_db.json";
+            container POLICIES {
+	            leaf inactivity_timeout {
+	                description "serial-console inactivity-timeout timer value in minutes";
+	                type int32 {
+	                    range "0..35000";
+	                }
+	                default 15;
```
Comment (qiluo-msft) [inline src/sonic-yang-models/yang-models/sonic-serial-console.yang:19]: >15 [](http://example.com/codeflow?start=25&length=2)

How do you pick the default value? Is it the linux default behavior in old image without this feature?

File: src/sonic-yang-models/yang-models/sonic-serial-console.yang [hunk 3]
```diff
@@ -0,0 +1,37 @@
+//filename:  sonic-serial-console.yang
+module sonic-serial-console {
```
Comment (qiluo-msft) [inline src/sonic-yang-models/yang-models/sonic-serial-console.yang:2]: >module [](http://example.com/codeflow?start=0&length=6)

@ganglyu Please help review the 2 Yang models.

Comment (ganglyu) [reply src/sonic-yang-models/yang-models/sonic-serial-console.yang:2]: @i-davydenko 
Can you create separate PR for sonic yang models?

File: src/sonic-yang-models/yang-models/sonic-serial-console.yang [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+	                description "serial-console inactivity-timeout timer value in minutes";
+	                type int32 {
+	                    range "0..35000";
+	                }
+	                default 15;
+	            }
+	
+	            leaf sysrq_capabilities {
+	                description "managing SysRq capabilities";
+	                type enumeration {
```
Comment (wen587) [inline src/sonic-yang-models/yang-models/sonic-serial-console.yang:24]: suggest to reuse `admin_mode` https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-templates/sonic-types.yang.j2#L127

File: src/sonic-yang-models/yang-models/sonic-ssh-server.yang (hunk 1 above)
Comment (i-davydenko) [reply src/sonic-yang-models/yang-models/sonic-ssh-server.yang:50]: @qiluo-msft hi, yes, this is old defaults in sonic

File: src/sonic-yang-models/yang-models/sonic-serial-console.yang (hunk 2 above)
Comment (i-davydenko) [reply src/sonic-yang-models/yang-models/sonic-serial-console.yang:19]: @qiluo-msft hi, yes, this is old defaults in sonic

File: src/sonic-yang-models/yang-models/sonic-serial-console.yang (hunk 4 above)
Comment (i-davydenko) [reply src/sonic-yang-models/yang-models/sonic-serial-console.yang]: @wen587 done

**Comment by qiluo-msft** [comment]
Please reduce the "force-pushed" usage, which invalidated all the previous review status.

**Review by qiluo-msft — DISMISSED** [review]

File: src/sonic-yang-models/yang-models/sonic-serial-console.yang (hunk 3 above)
Comment (liat-grozovik) [reply src/sonic-yang-models/yang-models/sonic-serial-console.yang:2]: @i-davydenko is there a PR for yang? can you please share?
also please add it to the HLD list of PRs for the feature

**Comment by i-davydenko** [comment]
@qiluo-msft not sure how to handle ms_conflict — MS conflict detect. Could you please advise what should I do?
Do I need to message someone from MS team?

**Comment by i-davydenko** [comment]
@StormLiangMS,@liushilongbuaa can you help please? What should I do with this ms_conflict run?
PR: https://github.com/sonic-net/sonic-buildimage/pull/17623 is conflict with MS internal repo

File: src/sonic-yang-models/yang-models/sonic-serial-console.yang (hunk 3 above)
Comment (qiluo-msft) [reply src/sonic-yang-models/yang-models/sonic-serial-console.yang:2]: This commnet is still active.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**